### PR TITLE
fix: configuration file

### PR DESCRIPTION
**Pull request by HJC5305**
1. modify the toolchain configuration file
2. modify control file

log: fix bug

### CODE REVIEW

**Comment by HJC5305** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by HJC5305** [comment]
I have read the CLA Document and I hereby sign the CLA.

File: src/scripts/toolchain.sh [hunk 1]
```diff
@@ -106,6 +106,7 @@ probe_build_systems()
 {
 	CMAKE_VERSIONS=($(find $search_path -name 'cmake'))
 	echo "{\"C/C++ build systems\": [" >> $TOOLCHAINS
+	CMAKE_VERSIONS=($CMAKE_VERSIONS)
```
Comment (deepin-mozart) [inline src/scripts/toolchain.sh:109]: 这里需要删除

File: src/scripts/toolchain.sh [hunk 2]
```diff
@@ -80,23 +80,23 @@ probe_cxx_compilers()
...
 probe_debuggers()
 {
-	DEBUGGERS=$(find $search_path -name gdb \
-			-o -name lldb-[1-9]*
+	DEBUGGERS=$(find /usr/bin -name gdb \
+			-o -name lldb-[1-9]* -o -name lldb
 		)
 	echo "{\"C/C++ debuggers\": [" >> $TOOLCHAINS
-	for debugger in ${DEBUGGERS[@]} lldb; do
+	DEBUGGERS=($DEBUGGERS)
```
Comment (deepin-mozart) [inline src/scripts/toolchain.sh:87]: 这里要改个名字

File: src/scripts/toolchain.sh [hunk 3]
```diff
@@ -80,23 +80,23 @@ probe_cxx_compilers()
 
 probe_debuggers()
 {
-	DEBUGGERS=$(find $search_path -name gdb \
-			-o -name lldb-[1-9]*
+	DEBUGGERS=$(find /usr/bin -name gdb \
```
Comment (deepin-mozart) [inline src/scripts/toolchain.sh:83]: 这里不用改为/usr/bin

**Review by deepin-mozart — CHANGES_REQUESTED** [review]
请按照评论进行修改

**Pull request closed without merging**